sentinel-extension/sentinel-prometheus-exporter/src/main/java/com/alibaba/csp/sentinel/prom/collector/SentinelCollector.java

```diff
@@ -0,0 +1,119 @@
+package com.alibaba.csp.sentinel.prom.collector;
```

Comment:
The package could be prefixed with `com.alibaba.csp.sentinel.metric.prom`.

Reply:
has changed.